tox.ini

```diff
@@ -20,7 +20,7 @@ norecursedirs = .* docs requirements site-packages
 [testenv]
 setenv = EDX_COOKIECUTTER_ROOTDIR = {toxinidir}
 deps =
-    -r{toxinidir}/requirements/test.txt
+    -r {toxinidir}/requirements/test.txt
```

Comment:
There was a bug in tox that used to prevent this from working; you've verified that it's fixed now?  I vaguely recall that it was, but not positive.